AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/Keyrings/RawRSAKeyring.dfy

```diff
@@ -162,10 +195,10 @@ module RawRSAKeyring {
       //= type=implication
       //# OnEncrypt MUST fail if this keyring does not have a specified [public
       //# key](#public-key).
-      ensures
-        this.publicKey.None? || |this.publicKey.Extract()| == 0
-        ==>
-          output.Failure?
+      // ensures
+      //   this.publicKey.None? || |this.publicKey.Extract()| == 0
+      //   ==>
+      //     output.Failure?
```

Comment:
Does this verify?
```suggestion
      ensures
        publicKeyMaterial.None?
        ==>
          output.Failure?
```
I see OnDecrypt made a similar change, I'd prefer to keep the statement associated with spec

Reply:
ensures restored